src/index.css

```diff
@@ -1,36 +1,37 @@
 /* Global App style rules */
 h1 {
-	font-size: 2.5rem;
-	font-weight: 200;
+  font-size: 2.5rem;
+  font-weight: 200;
 }
 
 h2 {
-	font-weight: 400;
+  font-weight: 400;
 }
 
 h3 {
-	font-weight: 400;
+  font-weight: 400;
 }
 
 body {
-	font-family: Helvetica, Arial, sans-serif;
+  font-family: Helvetica, Arial, sans-serif;
 }
 
 summary {
-	margin-bottom: .2em;
+  margin-bottom: .2em;
+  cursor: pointer;
```

Comment:
Oh, this was an unrelated change. I'll just leave it in here since it's small, but for some others I created a separate visual-fixes branch.